src/renderer/views/login/MiningView.tsx

```diff
@@ -46,6 +46,14 @@ const MiningView = observer(
       }
     }, [requirementEl.current]);
 
+    const requirement: string = useT(
+      "REQUIRE:\n" +
```

Comment:
Can we use `<br />` tag instead of `\n` even in near future?

Reply:
I really wish we could do that. Unfortunately React wouldn't allow that, although we could tweak Transifex Native a little bit to make it happen.